### PR TITLE
Add update KYC settings endpoint

### PR DESCRIPTION
**Pull request by swilczynski**

### CODE REVIEW

File: openapi/components/schemas/KycSettings.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+type: object
+description: The settings for score calculation.
+properties:
+  identity-proof:
+    $ref: ./KycDocument/Settings/KycSettingsIdentity.yaml
+  address-proof:
```
Comment (vykhrystiuk) [inline openapi/components/schemas/KycSettings.yaml:6]:
```suggestion
  addressProof:
```

**Review by removed-account-123 — APPROVED** [review]

**Comment by dariashka** [comment]
Just a small suggestion to rename the PR :) It looks more that we are adding `update KYC settings endpoint`.

**Review by removed-account-123 — APPROVED** [review]

**Comment by swilczynski** [comment]
@Rebilly/php please review

**Review by ashkarpetin — APPROVED** [review]

File: openapi/paths/kyc-settings.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+put:
+  x-products:
+    - Full
+    - Risk Management
+    - Core
+  tags:
+    - KYC Documents
+  summary: Update KYC settings
+  operationId: PutKycSettings
+  x-sdk-operation-name: kycSettings
```
Comment (VladimirBerdnik) [inline openapi/paths/kyc-settings.yaml:10]: I think this might be not the best name for the JS-SDK

**Pull request merged**